.gitignore

```diff
@@ -7,6 +7,7 @@
 
 # User-specific stuff:
 .idea/**/workspace.xml
+.idea/workspace.xml
```

Comment:
Shouldn't that be covered with the entry above?

Reply:
it's a mistake thanks.